etc/profile-m-z/rtv-addons.inc

```diff
@@ -0,0 +1,23 @@
+# This file is overwritten during software install.
+# Persistent customizations should go in a .local file.
+
+# You can configure rtv to open different type of links
+# in external applications. Configuration here:
+# https://github.com/michael-lazar/rtv#viewing-media-links
+# This include is meant to facilitate that configuration
+# with the use of a .local file.
+
+ignore nosound
+ignore private-bin
+ignore dbus-system none
+ignore dbus-user none
```

Comment:
can we use any specific dbus.user filter rules?

Reply:
i'm no expert, this is mostly my use case. eom brakes with "dbus-user none"

Reply:
yea, that didn't work for me either